lib/Conversion/TorchToLinalg/Reduction.cpp

```diff
@@ -277,6 +277,10 @@ static Value createInitElementForReduceOp(OpBuilder &b, Location loc,
   if (isa<AtenLinalgVectorNormOp>(op) || isa<AtenFrobeniusNormDimOp>(op))
     return b.create<arith::ConstantOp>(loc, b.getZeroAttr(elementType));
 
+  if (auto all = dyn_cast<AtenAllDimOp>(op)) {
```

Comment:
nit : use isa<AtenAllDimOp> 

to avoid compiler warnings

Reply:
Fixed